dex/testing/btc/harness.sh

```diff
@@ -8,21 +8,26 @@ export ALPHA_LISTEN_PORT="20575"
 export BETA_LISTEN_PORT="20576"
 export ALPHA_RPC_PORT="20556"
 export BETA_RPC_PORT="20557"
+export WALLET_PASSWORD="abc"
+# For a descriptor wallet (no seed), a [wallet_name]-desc.json file is used with
+# importdescriptors to set the wallet keys.
 export ALPHA_WALLET_SEED="cMndqchcXSCUQDDZQSKU2cUHbPb5UfFL9afspxsBELeE6qx6ac9n"
-export BETA_WALLET_SEED="cRHosJjgZ2UWsEAeHYYUFa8Z6viHYXm94GguGtpzMo6qwKBC1DSq"
 export ALPHA_MINING_ADDR="bcrt1qy7agjj62epx0ydnqskgwlcfwu52xjtpj36hr0d"
-export BETA_MINING_ADDR="bcrt1qsl5u7cvc7f96zewf5te6kzpunmd5plzuxgdlfq"
-export WALLET_PASSWORD="abc"
+export BETA_MINING_ADDR="bcrt1q8hzxze4zrjtpm27lrlanc20uhtswteye8ad7gx"
 # Gamma is a named wallet in the alpha wallet directory.
-export GAMMA_WALLET_SEED="cR6gasj1RtB9Qv9j2kVej2XzQmXPmZBcn8KzUmxSSCQoz3TqTNMg"
-export GAMMA_ADDRESS="bcrt1qh6m8v7czylaz8tzeaxxjqjhqgs0ruu0lu44ksy"
+export GAMMA_ADDRESS="bcrt1qt2xf2s5dkm4t34zm69w9my2dhwmnskwv4st3ly"
 # Delta is a named wallet in the beta wallet directory.
 export DELTA_WALLET_SEED="cURsyTZ8icuTHwWxSfTC2Geu2F6dMRtnzt1gvSaxHdc9Zf6eviJN"
 export DELTA_ADDRESS="bcrt1q4clywna5re22qh9mexqty8u8mqvhjh8cwhp5ms"
 export EXTRA_ARGS="--blockfilterindex --peerblockfilters --rpcbind=0.0.0.0 --rpcallowip=0.0.0.0/0"
 export CREATE_DEFAULT_WALLET="1"
-# Arguments are: createwallet "wallet_name" ( disable_private_keys blank(boolean) "passphrase" avoid_reuse descriptors load_on_startup external_signer )
+
+# The new-wallet script creates a new wallet with keys (not blank), no
+# descriptors and no passphrase.
+export NEW_WALLET_CMD="./\$1 -named createwallet wallet_name=\$2 descriptors=false load_on_startup=true"
+# Possible arguments are: "wallet_name" ( disable_private_keys blank(boolean) "passphrase" avoid_reuse descriptors load_on_startup external_signer )
 # $1 is the node to create with.
```

Comment:
Comments below the comand are for `NEW_WALLET_CMD` so can move above that or delete. Did not register you could pass arguments with `=` but that's much better.